src/components/NftPreview/NftPreview.tsx

```diff
@@ -79,21 +101,25 @@ const StyledNftPreview = styled.div<{ gap: number }>`
   }
 
   // use margin to create row-gap for now
-  @media only screen and ${breakpoints.mobile} {
-    width: 100%;
-    margin-bottom: 40px;
-  }
+  width: 100%;
+  margin-bottom: 40px;
 
-  // use margin to create row-gap for now
+  // width looks nasty but it allows us to conditionally apply different width rules based on # columns:
+  // - if single columm, use hardcoded width because the NFT isnt as wide as the whole page
+  // - if more columns, use calc to automatically set width based on column #
+  // this is important because while we *could* use hardcoded widths for desktop, we need to use dynamic widths for tablet
   @media only screen and ${breakpoints.mobileLarge} {
-    width: calc((100% - ${({ gap }) => gap * 3}px) / 3);
-    margin: ${({ gap }) => gap / 2}px;
+    width: ${({ columns }) => columns === 1
+    ? (SINGLE_COLUMN_NFT_WIDTH + 'px')
+    : 'calc((100% - ' + LAYOUT_GAP_BREAKPOINTS.mobileLarge * columns + 'px) / ' + columns + ')'};
```

Comment:
nit: we should be able to use backticks here since we're inside of a function, like
```
`${SINGLE_COLUMN_NFT_WIDTH}px`
```
so we don't have to sprinkle `+` everywhere

Reply:
nicee, it didn't work the first time I tried so I thought it was because we were already inside backticks. but just tried again and works 🤷🏻‍♂️